src/test/groovy/com/github/vkorobkov/jfixtures/sql/DataSourceUtil.groovy

```diff
@@ -0,0 +1,24 @@
+package com.github.vkorobkov.jfixtures.sql
+
+import org.apache.commons.dbcp2.BasicDataSource
+import spock.lang.Shared
+
+import javax.sql.DataSource
+
+class DataSourceUtil {
+    private static final String DATASOURCE_URL = "jdbc:h2:mem:jfixturesDb;DB_CLOSE_DELAY=-1;DATABASE_TO_UPPER=false"
+
+    private static DataSource dataSource
+
+    @Shared
+    public static groovy.sql.Sql sql = new groovy.sql.Sql(getDataSource())
```

Comment:
Take a look at `groovy.lang.Lazy` - probably we can just do something like:
```
@Lazy
static Sql sql = {
    def dataSource = new BasicDataSource()
    dataSource.url = DATASOURCE_URL
    new Sql(dataSource)
}()
```

or even:
```
@Lazy
static def sql = new Sql(new BasicDataSource(url: DATASOURCE_URL))
```